src/base.jl

```diff
@@ -1100,36 +1104,15 @@ See also: [`iterate_components`](@ref), [`get_components` with a filter](@ref ge
 ) where {T <: Component}),
 [`get_available_components`](@ref), [`get_buses`](@ref)
 """
-function get_components(
+function IS.get_components(
     ::Type{T},
     sys::System;
     subsystem_name = nothing,
 ) where {T <: Component}
     return IS.get_components(T, sys.data; subsystem_name = subsystem_name)
 end
 
-"""
```

Comment:
Moved to `get_components_interface` (though there's an argument it should appear here too)

Reply:
Not quite. `IS.get_components` will still be distinct from `PSY.get_components`, it's just that both will now be defined on `System` and the latter will just call the former. See the top of `get_components_interface.jl` for a detailed explanation.